### DISCUSSION TITLE
Scanning speed version 8.3.1

### DISCUSSION THREAD

**Discussion by JWBro** [LMS-Community/slimserver#839] [General]
Hi everyone,

I am interested how long it takes to scan a music collection from scratch.
My experiences:
On a HP Z620 workstation (2x Xeon CPU, 48 Gb mem) and a collection of ~5000 albums: roughly 30 minutes.
On an HP ElLitedesk 800 G1: a single CPU machine with i5-4590 and 8 Gb memory with 1600 albums: roughly 28 minutes.
NB both machines are running Windows 10 Pro (version 22H2, 64b). Both collection are all in FLAC format.

**Comment by tomscytale** [comment]
Disc read speed may as big a factor as CPU speed. 

The scanner is single process so more cores won't increase speed directly - they will however reduce contention from other processes for CPU time. 

Memory consumption doesn't appear to be much - a few hundred megs on my RPi (2GB)

**Reply by JWBro** [reply]
Disks on both systems have the SATA-600 interface and running at 7200 rpm.

**Comment by michaelherger** [comment]
Scan performance depends on a lot more than just the hardware. The same hardware will vary a lot in speed depending on the LMS configuration, plugins installed, operating system etc.
